docs/yaml.rst

```diff
@@ -78,73 +72,64 @@ Here is an example:
-
-    * **category** Splits data based on values assocatied with a BUFR mnemonic. Constists of:
-
-      * **variable** The variable from the **variables** section to split on.
-      * *(optional)* **map** Associates integer values in BUFR mnemonic data to a string. Please not
-        that integer keys must be prepended with an **_** (ex: **_2**). Rows where where the mnemonic
-        value is not defined in the map will be rejected (won't appear in output).
-* *(optional)* **filters** List of filters to apply to the data before exporting. Filters exclude data
-  which does not meet their requirements. The following filters are supported:
-
-  * **bounding**
-
-    * **variable** The variable from the *variables* section to filter on.
-    * *(optional)* **upperBound** The highest possible value to accept
-    * *(optional)* **lowerBound** The lowest possible value to accept
-
-.. note::
-    Either **upperBound**, **lowerBound**, or both must be present.
-
-Encoder Description
-~~~~~~~~~~~~~~~~~~~
-
-The **encoder** section defines the ObsGroup objects that will be created. Here is an example:
+``bufr`` keys
+^^^^^^^^^^^^^
+
+``group_by_variable`` *(optional)*
+    Name of a variable used to group observations when expanding repeated
+    sequences.
+
+``subsets`` *(optional)*
+    List of subset names to read.  When omitted all subsets matching the
+    queries are processed.
+
+``variables``
+    Mapping of arbitrary names to variable descriptions.  These names are later
+    referenced by the ``encoder`` section.  A variable description can be one of
+    the following:
+
+    - ``query`` – direct query into the BUFR tree.  Numeric results may apply
+      ``offset``, ``scale`` or ``wrap`` transforms and the type may be forced to
+      ``int``, ``int64``, ``float`` or ``double``.
+    - ``datetime`` – combine mnemonics for year, month, day, hour and minute
+      (and optionally second and hoursFromUtc) into an epoch time stored as
+      seconds since ``1970-01-01T00:00:00Z``.
+    - ``timeoffset`` – like ``datetime`` but the value is relative to a
+      ``referenceTime``.  Transforms may be used to convert units.
+    - specialised forms such as ``sensorScanAngle`` or
+      ``remappedBrightnessTemperature`` used in some satellite mappings.
+
+``splits`` *(optional)*
+    Splits divide the dataset into categories.  The ``category`` split type
+    partitions the data by the value of a variable and can map integer values to
+    string names.
+
+``filters`` *(optional)*
+    Filters remove rows prior to encoding.  The ``bounding`` filter keeps rows
+    whose values fall between ``lowerBound`` and ``upperBound`` (at least one of
+    these bounds must be supplied).
+
+Encoder section
+---------------
+
+The ``encoder`` section describes how the exported data should be written.  A
+shortened example is shown below:
 
 .. code-block:: yaml
 
   encoder:
+    type: netcdf
```

Comment:
Is it mandatory to add `type: netcdf` in the encoder part?

Reply:
Oh, I see.  The `type: netcdf` is optional.

Reply:
Thats a good point Emily. I went ahead and removed this field as it currently has no use. We can re-introduce it later if we find we have a need for it.